apps/docs/src/app/platform/component-docs/platform-list/platform-list-examples/platform-list-with-selection-example.component.ts

```diff
@@ -30,12 +30,21 @@ export class ListDataProvider extends DataProvider<Address> {
     selector: 'fdp-list-with-selection-example',
     templateUrl: './platform-list-with-selection-example.component.html'
 })
-export class PlatformListWithSelectionExampleComponent {
+export class PlatformListWithSelectionExampleComponent implements AfterViewInit {
     _dataSource = new ListDataSource<Address>(new ListDataProvider());
     _selectedItems: Address[] = [];
 
+    constructor(private _render: Renderer2, private _ele: ElementRef) {}
+
+    ngAfterViewInit(): void {
+        this._render.setAttribute( this._ele.nativeElement.querySelector('fd-toolbar'), 'tabindex', '0' );
+        this._render.setStyle( this._ele.nativeElement.querySelector('fd-toolbar'), 'outline', 'none' );
```

Comment:
By the way, why not just add this attribute and style to HTML on the "fd-toolbar" to avoid all this manipulation in component?

Reply:
Hi Yuri, because  inline styling is not suggested